spec/javascripts/live_search_spec.js

```diff
@@ -187,6 +174,42 @@ describe('liveSearch', function () {
     expect(liveSearch.cache('some-slug')).toBe('something in the cache')
   })
 
+  describe('the URL should change', function () {
+    beforeEach(function () {
+      // clear these options to simplify the URLs
+      $form.find('input[name=field]').prop('checked', false)
+      $form.find('input[name=published_at]').val('')
+    })
+
+    it('when the search options change', function () {
```

Comment:
Thanks for amending. This still reads wrong to me with "it when the search options change", which makes it not feel like an idiomatic way to express a scenario. Can it be changed so that the it forms the expectation? i.e. `it('should ...')`

Reply:
Okay, I've had another crack at it. Hopefully... better?